rakefile.rb

```diff
@@ -95,6 +101,9 @@
 desc "Execute default tasks"
 task :default => [ :vars, :gitlink, :unit, :integ, :spec, :pack ]
 
+desc "Execute default tasks plus .NET Core tasks"
+task :dotnet => [ :default, :unit_dotnet, :pack_dotnet]

```

Comment:
I don't think this is the right approach. We don't want to build the net40 solution and then the dotnet solution. We want to build the dotnet solution only. It should really be the `dotnet` analog of the `default` task, i.e.

```ruby
+task :dotnet => [ :vars, :gitlink_dotnet, :unit_dotnet, :integ_dotnet, :spec_dotnet, :pack_dotnet ]
```